tools/partitions/insights.csv

```diff
@@ -0,0 +1,5 @@
+# Name,   Type, SubType, Offset,  Size, Flags
+nvs,       data,  nvs,     0x9000,   0x5000,
+otadata,   data,  ota,     0xe000,   0x2000,
+ota_0,     app,   ota_0,   0x10000,  0x1E0000,
+coredump, data, coredump, , 64K,
```

Comment:
we should not need this anymore?

Reply:
Have you tested any Insights examples with other partitions? If it works well, then this can be removed.

Reply:
I have not tried because look at my previous comment: "if you do not lower the size of the last partition, coredump will end up outside of the flash chip"